init.lua

```diff
@@ -234,9 +252,14 @@ minetest.register_on_leaveplayer(function(player)
 	if player_lights[name] then
 		remove_light(player_lights[name].pos)
 	end
-	player_lights[name] = nil
+	goodtorch.player_lights[name] = nil
 end)
 
+local water_sound = default.node_sound_water_defaults()
+if goodtorch.detect_gamemode() == "MCL" then
+	water_sound = mcl_sounds.node_sound_water_defaults()
+end
+
```

Comment:
What the, that's wrong, that will crash when we attempt to join a non-default game (Like MineClone).

```
-- My line 258 to 263
local water_sound = nil
if goodtorch.detect_gamemode() == "MTG" then
	water_sounds = default.node_sound_water_defaults()
elseif goodtorch.detect_gamemode() == "MCL" then
	water_sound = mcl_sounds.node_sound_water_defaults()
end
```

Reply:
On further investigation I still don't see that in my fork of this repo... so I will hard change it in my code via the web and manually attempt to commit it.

Reply:
Hmm, I'm now seeing my repo has this change so I guess I picked the wrong commit (this is a after the fact, before my fix)

Reply:
[Heres the fix](https://github.com/LissoBone/goodtorch/pull/2#pullrequestreview-1087789778) further down the page. (for those who love scrolling further down)

Reply:
I have just exited my refridgerator. Time for me to carefully analyze all your changes.